tests/gpflow/utilities/test_multipledispatch.py

```diff
@@ -67,9 +68,19 @@ def test_a2_b2(x, y):
         assert len(w) == 0
 
 
+dispatcher_expectautographwarning_combinations_to_test = [
+    (gpflow.utilities.Dispatcher, False),  # no warnings with our custom Dispatcher
+]
+
+if Version(tf.__version__) < Version("2.3"):
```

Comment:
Not sure what our policy is, but do we really want to have backwards compatible tests? I'd be happy with staying up to date with TF 2.3.

Reply:
not in general necessarily, but in this case as it's very easy to write a test that passes because the test is written wrong, I wanted to keep the original test in there ...

Reply:
but on second thought it might be cleaner to simply skip the test when tf.version >= 2.3 ...